scripts/PearlPowderISIS/pearl_routines.py

```diff
@@ -341,7 +340,14 @@ def PEARL_focus(number, ext="raw", fmode="trans", ttmode="TT70", atten=True, van
     return outwork
 
 
-def PEARL_focus_v1(number, ext="raw", fmode="trans", ttmode="TT70", atten=True, van_norm=True, debug=False):
+def pearl_run_focus(number, ext="raw", fmode="trans", ttmode="TT70", atten=True, van_norm=True, debug=False,
+                    focus_mode=1):
+    alg_range = 12
+    save_range = 3
+    if focus_mode == 2:
+        alg_range = 14
+        save_range = 5
+
```

Comment:
Aha interesing, so this was all the difference between focus_v1 and focus_v2?

@shahrozahmed1: I see this is "in progress". I skimmed through the code changes and it looks good. Let me know when #16459 and this are ready for testing.